imitation/scripts/data_collect.py

```diff
@@ -18,15 +21,91 @@ def make_PPO2(env_name, num_vec, **make_blank_policy_kwargs):
 
 
 @data_collect_ex.main
-def main(env_name, total_timesteps, num_vec=8, make_blank_policy_kwargs={}):
+def main(env_name, total_timesteps,
```

Comment:
May as well add type annotations to keyword arguments 24-27 given you do for the rest.